### ISSUE TITLE
stop() function in Inline assembly calls is treated as solidity functions

### ISSUE THREAD

**Issue by vanruch** [crytic/slither#742]
When trying to analyze contract that contains code like this

```
assembly {
  ...
  stop()
  ...
}
```
I'm getting the following error when `name` = `stop()`.
```  
File "/home/ethsec/.local/lib/python3.6/site-packages/slither/core/declarations/solidity_variables.py", line 154, in __init__
    assert name in SOLIDITY_FUNCTIONS
```

Maybe `stop` should be added to `evm_functions.py` list?

**Comment by montyly** [comment]
Hi @vanruch, thank you for reporting this. We will fix it

**Comment by montyly** [comment]
Hi @vanruch 

We added the support in master. The fix will be available in the upcoming [0.7.1](https://github.com/crytic/slither/issues/789) release
